templates/zerver/help/send-a-private-message.md

```diff
@@ -4,29 +4,35 @@ Zulip allows users to send private messages to each other.
 
 ## There are two ways to send a Private Message:
 
-### Way 1:
+### By using the the message box.
 
-* To submit a new private message, first click on the `New Private
-  Message` button located at the bottom of your screen.
+1. Click on the `New Private Message` button located at the bottom of your
+screen. The message box, like the one shown below will appear at the bottom
+center of the screen.
+  ![New stream message](/static/images/help/private-box.png)
 
-* The following box will be displayed
-![New stream message](/static/images/help/private-box.png)
+2. In the field labeled **one or more people...**, input the username or email
+address of the user you want to privately message. When you begin typing an
+email address or username, autocomplete suggestions will appear.
 
-* In the **one or more people...** field, input the username or email
-address of the user you want to privately message.  When you begin
-typing an email address or username, autocomplete suggestions will
-appear.
+    !!! warn ""
+        If you attempt to input a nonexistent username or email address, an
+        error message will appear and the message will not be delivered.
 
-* If you attempt to input a nonexistent username or email address, an
-  error message will appear.
+3. Enter your message in the box labeled **Compose your message here...**. You
+can now send your private message.
 
-### Way 2:
+### By selecting user from **USERS** column.
 
-* You also can click on the **user's name** in the `USERS` panel on
-the right and click on `down arrow` that gets visible when hovering
-over the name and click on the option `Compose a private message to
-<user's name>` and write the message in the compose box.
+1. You also can click on the **user's name** in the **USERS** panel on
```

Comment:
We don't use "USERS panel"; instead, we use "the user list on the right sidebar"